src/components/NavHeader.js

```diff
@@ -0,0 +1,42 @@
+import { Link } from 'react-router-dom';
+import '../styles/NavHeader.css';
+import React from 'react';
+// import { NavLink } from 'react-router-dom';
+// import planetImg from '../public/planet2.png';
+
+const NavHeader = () => {
+  const urls = [
+    {
+      id: 1,
+      url: '/rocket',
+      text: 'Rocket',
+    },
+    {
+      id: 2,
+      url: '/mission',
+      text: 'Missions',
+    },
+    {
+      id: 3,
+      url: '/Profile',
+      text: 'My Profile',
+    },
+  ];
+
+  return (
+    <nav className="navbar">
+      <img src="3212580.png" className="PlantImg" alt="Planet-img" />
+      <h1 className="nav-heading">Space Travelers Hub</h1>
+
+      <ul className="nav-main">
+        {urls.map((url) => (
+          <li className="nav-items" key={url.id}>
+            <Link to={url.url}>{ url.text }</Link>
```

Comment:
- [ ] it would be better to use `<NavLink/>` to create navigation links and make sure to style the one that is clicked (active) as required 
- screenshot of the requirements 
<img width="959" alt="Screenshot 2023-04-13 at 17 26 46" src="https://user-images.githubusercontent.com/61790587/231838159-5db15906-e910-4c3a-ae5b-7f5b72d0f51f.png">